docs/source/index.rst

```diff
@@ -1,50 +1,42 @@
 .. Composer documentation master file
 
-Composer (Beta)
-===============
+Composer
+========
 
-MosaicML ``Composer`` contains a library of ML training efficiency methods, and a modular approach to compose them
-together to train deep neural networks. We aim to ease the transition from research to industry through reproducible
-code and rigorous benchmarking. With Composer, speed-up or accuracy-boosting methods can be easily composed into
-complete recipes.
+Composer provides well-engineered implementations of efficient training methods
+to give the tools that help you train a better model for cheaper.
 
-The library features:
+Using Composer, you can:
 
-* Implementation of 20+ efficiency methods curated from the research community
-* Standardized approach to implement and compose efficiency methods, extended from two-way callbacks
-  (`Howard et al, 2020 <https://arxiv.org/abs/2002.04688>`_)
-* Easy way to access our methods either directly for your trainer loops, or through the Composer trainer
+- Train an ImageNet model to 76.1% accuracy for $37 (`with vanilla PyTorch`:$127)
+- Train a GPT-2 125M to a perplexity of 23.9 for $148 (`with vanilla PyTorch`: $255)
+- Use start-of-the-art implementations of methods to speed up your own training loop.
 
-.. note::
-    MosaicML Composer is currently in **beta**, so the API is subject to change.
+At MosaicML, we are focused on making training ML models accessible. To do this,
+we continually productionize state-of-the-art academic research on efficient model
+training, and also study the `combinations`` of these methods in order to ensure
+that model training is ✨ as efficient as possible ✨.
 
 
-Motivation
-~~~~~~~~~~
+Composer features:
 
-MosaicML exists to make ML training more efficient. We believe large scale ML should be available to everyone
-not just large companies.
+- 20+ efficient training methods for training a better language and vision models! Don't waste hours trying to reproduce research papers when Composer has done the work for you.
+- Easy-to-use (optional) Trainer interface written to be as performant as possible, and `integrated best practices<https://www.mosaicml.com/blog/best-practices-dec-2021>`.
```

Comment:
[nit] Remove "optional".  As written it makes it sound like the "Easy-to-use" part is optional... but also we want people to use our Trainer so let's actively encourage it